src/stan/error_handling/matrix/check_pos_definite.hpp

```diff
@@ -46,8 +47,9 @@ namespace stan {
         std::ostringstream msg;
         msg << "is not positive definite. " 
                 << name << "(0,0) is ";
+        std::string msg_str(msg.str());
```

Comment:
There's no need to define an intermediate msg_str here that I can see.  Not that it's a big deal.